roles/multi-tenancy/tasks/tenant-cas-customizations.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- name: update tenant cas customizations
+  block:
+    - name: tenant cas - MPP workers
+      command:
+        cmd: "{{ DEPLOY_DIR }}/site-config/create-cas-server.sh --tenant {{ tenant | lower }} \
+             --output {{ DEPLOY_DIR }}/site-config \
+             --workers {{ settings.worker_count if (settings.worker_count is defined and settings.worker_count is not none ) else 0 }} \
+             --backup {{ 1 if (settings.backup_controller_enabled is defined) and \
+                              (settings.backup_controller_enabled|bool == True) else 0 }}"
+
+    - name: tenant cas user-defined resources - copy template
+      copy:
+        src: "{{ role_path }}/templates/cas-manage-cpu-and-memory.yaml"
+        dest: "{{ DEPLOY_DIR }}/site-config/cas-{{ tenant | lower }}-default/cas-manage-cpu-and-memory.yaml"
```

Comment:
In the future it may be easier to set a fact that contains the tenant or variable in its final state so here you could have `_tenant` set to `{{ tenant | lower }}` which would only require the conversion once.

Reply:
Updated in the latest change. Instead of setting a fact, I added conversion when key is passed to the task. So now lower conversion only occurs at one place.